go/types/blob.go

```diff
@@ -30,10 +30,9 @@ func NewEmptyBlob() Blob {
 }
 
 // BUG 155 - Should provide Write... Maybe even have Blob implement ReadWriteSeeker
-func (b Blob) Reader() io.ReadSeeker {
-	iter := newSequenceIterator(b.seq, 0)
-	return &BlobReader{b.seq, iter, nil, 0}
-
+func (b Blob) Reader() *BlobReader {
```

Comment:
Note that blob.go was (a) returned to its state prior to commit of original go read ahead and (b) added `BlobReader.Copy`